XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -326,6 +326,8 @@ public Vector2 NudgeConstraints
 
         private bool isRotationPossible = false;
 
+        private Vector3 offsetPosition = Vector3.zero;
```

Comment:
Can these properties not be public?  These seem like something you may want to change on a per-object basis?

Reply:
No, this is calculated at runtime by the distance between the grabbed position and the objects position to prevent "snapping" to the pointers current position. 

This instead makes the model move from the offset of the start of the grab.